File: SubscriptionUi/Subscription.Ui.Mvc/Angular/ReturnType/base-return-type.ts
class baseReturnType<T> {
    constructor() { }
    hasException: boolean;
    exception: string;
    result: T
}

class baseListReturnType<T>{

    entityList: T;

    totalCount: number;

}

class baseGlobalReturnType {
    errorMessage: string
    hasError: boolean;
}

class baseResultReturnType<T> {
    status: STATUS_MESSAGE;
    errorMessage: string;
    id: string;
    result: T
}

File: SubscriptionBackEnd/Subscription.Business/ReturnType/BaseReturnType.cs
using Subscription.Business.Enums;

namespace Subscription.Business.ReturnType
{
    public class BaseReturnType<T>
    {
        public T Result { get; set; }
        public RequestStatusEnum Status { get; set; }
        public string ErrorMessage { get; set; }
        public string Id { get; set; }
    }
}